tests/unit/backend/layers/api/test_curation_api.py

```diff
@@ -679,7 +678,6 @@ def test__get_collection_verify_body_is_reshaped_correctly__OK(self, mock_config
                 "datasets": [expect_dataset],
                 "doi": None,
                 "links": links,
-                "processing_status": "PENDING",
```

Comment:
why are you removing this? It seems to me that we shouldn't be removing any test lines where the processing status was `"PENDING"` or `"INITIALIZED"` because that by definition means the dataset wasn't published, and so the processing status should be reported.

Reply:
because this test only tests a published collection is returned correctly and published collections don't return a processing_status. `test_get_collection_dynamic_fields` tests for processing status in all of the other cases.

Reply:
Yeah I guess what I mean is how did we previously have a _passing test_ that included `"processing_status": "PENDING"` in the `expected_body` when one of the asserts is `self.assertEqual(expected_body, res_body)`. 

Does that mean we were permitting publication of Collections with Datasets that did not have a processing statue of `SUCCESS`?? Or is there a nuance here in the "publication" of test Datasets that allows bypassing of that check, which then meant this test would pass? Because this test appears to have been for a _published_ Collection (version).

My understanding is that we were errantly returning this attribute for published Collections, and NOT that some published Collections had Datasets with statuses other than `SUCCESS`.

Reply:
In the old test case we only cared that a processing_status was in the response, not what the actually value is. This is a nuance of the test. In the wild the frontend would gate keep us from publishing anything with datasets that are erroneous or processing.

Reply:
remember that only the frontend permits publishing of a collection.

Reply:
Ok, thanks for clarifying

>remember that only the frontend permits publishing of a collection.

Right, this wasn't a concern.